ocaml/idl/datamodel.ml

```diff
@@ -2274,6 +2283,18 @@ module Tunnel = struct
         ; field ~ty:tunnel_protocol ~default_value:(Some (VEnum "gre"))
             ~lifecycle:[(Published, "1.250.0", "Add protocol field to tunnel")]
             "protocol" "The protocol used for tunneling (either GRE or VxLAN)"
+        ; field ~qualifier:StaticRO ~ty:Int ~default_value:(Some (VInt 0L))
+            ~lifecycle:[(Published, rel_next, "Set vxlan id to tunnel")]
+            "tunnel_id" "VXLAN network identifier of tunnel"
+        ; field ~qualifier:StaticRO ~ty:Bool ~default_value:(Some (VBool false))
+            ~lifecycle:
+              [
+                ( Published
+                , rel_next
+                , "Set true if it is cross-server-private-network"
```

Comment:
Same problem: "set" is the wrong word: "true for a cross-server private network (observe where to put hyphens"